models/staging/src_fivetran_log.yml

```
@@ -135,6 +135,8 @@ sources:
       - name: account_membership
         identifier: "{{ var('fivetran_log_account_membership_identifier', 'account_membership')}}"
         description: Table of the users given access to this account.
+        meta:
```

@fivetran-jamie actually did some digging and found that dbt-core now has a [feature built in](https://docs.getdbt.com/reference/source-configs) to allow for the dynamic disabling of source! 

This means we don't need to use our override of the freshness macro and instead just disable the source freshness altogether! I just tested the new config and it seems to work as intended. 

Would you be able to try on your end the following update for all of the sources you modified to see if it works on your end as well?
```suggestion
        config:
```

---

@fivetran-joemarkiewicz requested changes have been made 😎